src/main/histFileMod.F90

```diff
@@ -289,7 +315,7 @@ end subroutine copy_entry_interface
   !
   character(len=max_length_filename) :: locfnh(max_tapes)  ! local history file names
   character(len=max_length_filename) :: locfnhr(max_tapes) ! local history restart file names
-  logical :: htapes_defined = .false.            ! flag indicates history contents have been defined
+  logical :: htapes_defined = .false.        ! flag indicates history output fields have been selected
```

Comment:
The wording of "defined" is specific here, because it's a term used in setting up NetCDF files. I like your switch of contents to "output fields", but I would keep defined over selected.